products/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+from django.db.models.aggregates import Count
+from django.db.models.deletion import CASCADE
+from django.db.models.fields import IntegerField
+from django.db.models.fields.related import OneToOneField
+
+class MainCategory(models.Model):
+    name       = models.CharField(max_length=50)
+    created_at = models.DateTimeField(auto_now_add=True)
+    
+    class Meta:
+        db_table = 'maincategories'
+
+class SubCategory(models.Model):
+    name = models.CharField(max_length=50)
+    maincategory = models.ForeignKey(MainCategory, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'subcategories'
+
+class Product(models.Model):
+    sub_category = models.ForeignKey(SubCategory, on_delete=models.CASCADE)
+    name         = models.CharField(max_length=45, unique=True)
+    price        = models.DecimalField(max_digits=10, decimal_places=3)
+    body         = models.URLField(max_length=200)
+    material     = models.CharField(max_length=100)
+    data         = models.DateField()
+    discount     = models.FloatField(default=1)
+    like         = models.BooleanField(default=False)
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now_add=True)
+    is_deleted   = models.BooleanField(default=False)
+    
+    class Meta:
+        db_table = 'products'
+
+class Image(models.Model):
+    image_url = models.URLField(max_length=200)
```

Comment:
동일한 단어의 반복은 피해주셔도 좋습니다!
```suggestion
    url = models.URLField(max_length=200)
```

Reply:
넵 수정하였습니다.